### PR TITLE
added company - Atento

### PR DESCRIPTION
**Pull request by shailendrakawadkar**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [ ] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by shailendrakawadkar** [comment]
@KieranRobson added new company from list, please verify

File: company-profiles/atento.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ - Pleasant Grove, UT, United States
+ - Buenos Aires, Buenos Aires Province, Argentina
+
+  
+
+##  How to apply
+
+  
+
+[Click Here to apply](https://atento.gupy.io/)
```
Comment (KieranRobson) [inline company-profiles/atento.md:62]:
```suggestion
##  How to apply
[Click Here to apply](https://atento.gupy.io/)
```

File: company-profiles/atento.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  
+
+##  Office locations
+
+  
+
+ - Miramar, FL, United States
+ - San Antonio, TX, United States
+ - Pleasant Grove, UT, United States
+ - Buenos Aires, Buenos Aires Province, Argentina
```
Comment (KieranRobson) [inline company-profiles/atento.md:54]:
```suggestion
 - Miramar, FL, United States
 - San Antonio, TX, United States
 - Pleasant Grove, UT, United States
 - Buenos Aires, Buenos Aires Province, Argentina
```

File: company-profiles/atento.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+  
+Brazil, Mexico and EMEA
+
+  
+
+##  Company technologies
+
+  
+Our unique expertise lies in combining data technology and digital tools with decades of experience in customer experience. This allows us to provide valuable insights for every step of the end/customer's journey, resulting in more valuable experiences for consumers and companies.
```
Comment (KieranRobson) [inline company-profiles/atento.md:44]:
```suggestion
##  Company technologies
Our unique expertise lies in combining data technology and digital tools with decades of experience in customer experience. This allows us to provide valuable insights for every step of the end/customer's journey, resulting in more valuable experiences for consumers and companies.
```

File: company-profiles/atento.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  
+
+You will be working from home on an Atento PC, so there is an expectation that you are organized and self driven.
+
+  
+
+##  Region
+
+  
+Brazil, Mexico and EMEA
```
Comment (KieranRobson) [inline company-profiles/atento.md:37]:
```suggestion
##  Region
Brazil, Mexico and EMEA
```

File: company-profiles/atento.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+92,663 employees on LinkedIn
+
+  
+
+##  Remote status
+
+  
+
+You will be working from home on an Atento PC, so there is an expectation that you are organized and self driven.
```
Comment (KieranRobson) [inline company-profiles/atento.md:30]:
```suggestion
##  Remote status
You will be working from home on an Atento PC, so there is an expectation that you are organized and self driven.
```

File: company-profiles/atento.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+Our mission is to help unleash the potential of every team.
+
+  
+
+##  Company size
+
+  
+
+92,663 employees on LinkedIn
+
```
Comment (KieranRobson) [inline company-profiles/atento.md:23]:
```suggestion
##  Company size
92,663 employees on LinkedIn

```

File: company-profiles/atento.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
+#  Atento
+
+  
+
+##  Company blurb
+
+  
+
+Over 200,000 people trust Atlassian.
```
Comment (KieranRobson) [inline company-profiles/atento.md:9]:
```suggestion
##  Company blurb
Over 200,000 people trust Atlassian.
```

File: company-profiles/atento.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
+#  Atento
+
+  
+
+##  Company blurb
```
Comment (KieranRobson) [inline company-profiles/atento.md:5]:
```suggestion
#  Atento

##  Company blurb
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Clean up of white space

File: company-profiles/atento.md (hunk 2 above)
Comment (shailendrakawadkar) [reply company-profiles/atento.md]: removed

File: company-profiles/atento.md (hunk 3 above)
Comment (shailendrakawadkar) [reply company-profiles/atento.md]: removed

File: company-profiles/atento.md (hunk 4 above)
Comment (shailendrakawadkar) [reply company-profiles/atento.md]: removed

**Comment by shailendrakawadkar** [comment]
@KieranRobson  removed spaces

**Comment by KieranRobson** [comment]
Logs arent loading and I cannot see why the tests are failing. 

Give me a bit of time before reviewing to see if they load.

File: README.md [hunk 9]
```diff
@@ -66,6 +66,7 @@ Name | Website | Region
 [Articulate](/company-profiles/articulate.md) | https://www.articulate.com | Worldwide
 [Astronomer](/company-profiles/astronomer.md) | https://www.astronomer.io/ | USA
 [Atlassian](/company-profiles/atlassian.md) | https://www.atlassian.com/ | USA, Europe, Asia, Australia
+[Atento](/company-profiles/atento.md) | https://atento.gupy.io | Brazil, Mexico and EMEA
```
Comment (KieranRobson) [inline README.md:69]:
```suggestion
[Atento](/company-profiles/atento.md) | https://atento.gupy.io | Brazil, Mexico and EMEA
[Atlassian](/company-profiles/atlassian.md) | https://www.atlassian.com/ | USA, Europe, Asia, Australia
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Quick change to be made

File: README.md (hunk 9 above)
Comment (shailendrakawadkar) [reply README.md]: fixed ordering

**Comment by shailendrakawadkar** [comment]
@KieranRobson fixed ordering, please check.

**Pull request merged**